PG16 compile failure

Getting this error when compiling against PG16

```
ogr_fdw.c: In function ‘ogrLookupGeometryFunctionOid’:
ogr_fdw.c:1562:10: error: too few arguments to function ‘stringToQualifiedNameList’
 1562 |  names = stringToQualifiedNameList(proname);
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
In file included from ogr_fdw.c:34:
/usr/include/postgresql/16/server/utils/regproc.h:28:14: note: declared here
   28 | extern List *stringToQualifiedNameList(const char *string, Node *escontext);
      |              ^~~~~~~~~~~~~~~~~~~~~~~~~
make: *** [<builtin>: ogr_fdw.o] Error 1

```
Also is there a reason you have ci turned off on master branch?

--- a/ogr_fdw.c
+++ b/ogr_fdw.c
@@ -1559,7 +1559,11 @@ ogrLookupGeometryFunctionOid(const char* proname)
 		return InvalidOid;
 	}
 
+#if PG_VERSION_NUM < 160000
 	names = stringToQualifiedNameList(proname);
+#else
+	names = stringToQualifiedNameList(proname,NIL);
+#endif
 #if PG_VERSION_NUM < 90400
 	clist = FuncnameGetCandidates(names, -1, NIL, false, false);
 #elif PG_VERSION_NUM < 140000

--- a/ogr_fdw.c
+++ b/ogr_fdw.c
@@ -1562,7 +1562,7 @@ ogrLookupGeometryFunctionOid(const char* proname)
 #if PG_VERSION_NUM < 160000
 	names = stringToQualifiedNameList(proname);
 #else
-	names = stringToQualifiedNameList(proname,NIL);
+	names = stringToQualifiedNameList(proname,NULL);
 #endif
 #if PG_VERSION_NUM < 90400
 	clist = FuncnameGetCandidates(names, -1, NIL, false, false);

